libs/island-ui/core/src/lib/Navigation/Navigation.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+describe('Navigation', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<div>ok</div>)
```

Comment:
this is questionable 🧐 
maybe remove useless test or try to render the navigation 🙂

Reply:
lol didn't notice this! i will remove it